docs/scos/dev/feature-integration-guides/202200.0/glue-api/glue-api-request-for-quote-feature-integration.md

```diff
@@ -0,0 +1,647 @@
+|---------------------|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Spryker Core API    | {{page.version}} | [Glue API: Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/glue-api-spryker-core-feature-integration.html) |
+| Cart API            | {{page.version}} | [Glue API: Cart feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/glue-api-cart-feature-integration.html)                 |
+| Company Account API | {{page.version}} | [Glue API: Cart feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/glue-api-company-account-feature-integration.html)      |
+| Quotation Process   | {{page.version}} | [Quotation process feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/quotation-process-feature-integration.html)                   |
+
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker/quote-requests-rest-api:"^0.1.4" spryker/quote-request-agents-rest-api:"^0.1.4" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following modules have been installed:
+
+| MODULE                        | EXPECTED DIRECTORY <!--for public Demo Shops-->  |
+|-------------------------------|--------------------------------------------------|
+| QuoteRequestAgentsRestApi     | vendor/spryker/quote-request-agents-rest-api     |
+| QuoteRequestsRestApi          | vendor/spryker/quote-requests-rest-api           |
+| QuoteRequestsRestApiExtension | vendor/spryker/quote-requests-rest-api-extension |
+
+{% endinfo_block %}
+
+### 2) Set up the configuration
+
+Add following configuration to your project configuration:
+
+| CONFIGURATION                                     | SPECIFICATION                                                    | NAMESPACE                        |
+|---------------------------------------------------|------------------------------------------------------------------|----------------------------------|
+| CompanyUsersRestApiConfig::COMPANY_USER_RESOURCES | Specifies resources which are accessible only for company users. | Spryker\Glue\CompanyUsersRestApi |
+
+```php
+<?php
+
+namespace Pyz\Glue\CompanyUsersRestApi;
+
+use Spryker\Glue\CompanyUsersRestApi\CompanyUsersRestApiConfig as SprykerCompanyUsersRestApiConfig;
+use Spryker\Glue\QuoteRequestsRestApi\QuoteRequestsRestApiConfig;
+
+class CompanyUsersRestApiConfig extends SprykerCompanyUsersRestApiConfig
+{
+    /**
+     * @var array<string>
+     */
+    protected const COMPANY_USER_RESOURCES = [
+        QuoteRequestsRestApiConfig::RESOURCE_QUOTE_REQUESTS,
```

Comment:
don't we need to mention `src/Pyz/Glue/AgentAuthRestApi/AgentAuthRestApiConfig::getAgentResources` as well?

```
QuoteRequestAgentsRestApiConfig::RESOURCE_AGENT_QUOTE_REQUESTS,
QuoteRequestAgentsRestApiConfig::RESOURCE_AGENT_QUOTE_REQUEST_SEND_TO_CUSTOMER,
QuoteRequestAgentsRestApiConfig::RESOURCE_AGENT_QUOTE_REQUEST_REVISE,
QuoteRequestAgentsRestApiConfig::RESOURCE_AGENT_QUOTE_REQUEST_CANCEL,
```

Reply:
No, because you need an agent access token to access these resources, you can't access it with a regular customer token.